ckanext/dcatapchharvest/harvesters_i14y.py

```diff
@@ -0,0 +1,206 @@
+import json
+
+import ckan.plugins as p
+import ckan.model as model
+from ckan.logic import NotFound, get_action
+
+from ckanext.dcat.harvesters.rdf import DCATRDFHarvester
+from ckanext.dcat.interfaces import IDCATRDFHarvester
+from ckanext.dcatapchharvest.dcat_helpers import get_pagination
+from ckanext.dcatapchharvest.harvest_helper import (
+    map_resources_to_ids,
+    check_package_change,
+    create_activity,
+)
+
+import logging
+log = logging.getLogger(__name__)
+
+
+class SwissDCATI14YRDFHarvester(DCATRDFHarvester):
+    p.implements(IDCATRDFHarvester, inherit=True)
+
+    harvest_job = None
+    current_page_url = None
+
+    def info(self):
+        return {
+            'name': 'dcat_ch_i14y_rdf',
+            'title': 'DCAT-AP Switzerland I14Y RDF Harvester',
+            'description': 'Harvester for DCAT-AP Switzerland datasets from an RDF graph desighned for i14Y'  # noqa
+        }
+
+    def validate_config(self, source_config):
+        source_config = super(SwissDCATI14YRDFHarvester, self).validate_config(source_config)  # noqa
```

Comment:
Could you please break lines to make them shorter rather than adding `# noqa`? It really makes it easier to read imo.

Reply:
done